typing/includemod.ml

```diff
@@ -193,15 +193,15 @@ and print_coercion3 ppf (i, n, c) =
 
 (* Simplify a structure coercion *)
 
-let simplify_structure_coercion cc id_pos_list =
+let simplify_structure_coercion runtime_fields cc id_pos_list =
   let rec is_identity_coercion pos = function
```

Comment:
can we move `runtime_fields` into the last argument to make it more consistent

Reply:
sure